neptune/new/internal/operation.py

```diff
@@ -553,3 +554,30 @@ def resolve(self, backend: "NeptuneBackend") -> Operation:
             backend, self.container_id, self.container_type, self.source_path
         )
         return create_assignment_operation(self.path, value)
+
+
+@dataclass
+class ChangeStage(Operation):
+    container_id: str
+    stage: str
```

Comment:
I think the internal state could be `ModelVersionStage` and the `value` of it could be used only in serialization/deserialization + the constructor will be the default for dataclass